src/build/config/config.yaml

```diff
@@ -867,6 +867,23 @@ option:
       restore: {}
       verify: {}
 
+  limit-time:
```

Comment:
`--storage-at-time`?  `--storage-pit`?  `--storage-snapshot`? Not really sure what the term of art is here, but might be worth a few more naming suggestions.

Reply:
I'm not that excited about limit-time but not sure I like any of these options better. Is `repo-limit-time` any better?

Reply:
`repo-snapshot-time`?

Reply:
`repo-snapshot-time` sounds the best to me so far.

Reply:
if possible, I'd prefer to avoid using `snapshot here. While I understand why it might seem appropriate (viewing the repository as a capture at a specific point in time), I believe it could confuse users, as it could easily suggest that we support filesystem snapshots, a term widely used in sysadmin tasks :worried:

Reply:
I think I agree with Stefan about `snapshot`. How about `repo-target-time`?

Reply:
Yeah, that works for me, and seems a little clearer.

Reply:
Would work for me too :+1:

Reply:
Name change done here: 548ebfba014

Reply:
Reviewed that commit in isolation and it looked good (not on a computer so didn’t check for anything missed). Thanks!